core/src/main/java/org/apache/carbondata/core/datamap/DistributableDataMapFormat.java

```diff
@@ -311,6 +314,13 @@ public boolean isFallbackJob() {
     return isFallbackJob;
   }
 
+  /**
+   * @return Whether asyncCall to the IndexServer.
+   */
+  public boolean isAsyncCall() {
```

Comment:
remove comment from here, and add proper comment to veriable declaration

Reply:
done